compiler/cpp/src/thrift/generate/t_go_generator.cc

```diff
@@ -1847,6 +1860,56 @@ void t_go_generator::generate_go_struct_writer(ostream& out,
   }
 }
 
+void t_go_generator::generate_go_struct_deepequal(ostream& out,
+                                               t_struct* tstruct,
```

Comment:
nit: I'm not sure what's the coding style for compiler c++ code but this indentation looks weird.

Reply:
Well, I use clang-format according to https://github.com/apache/thrift/blob/master/.clang-format